src/seedu/addressbook/commands/FindIgnoreCaseCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case sensitive.
```

Comment:
Remember to update your comments to accurately reflect the intended functionality.